### ISSUE TITLE
Analyse "Yes/No" / "Yes" / checkbox treatment of boolean entries

### ISSUE THREAD

**Issue by matkoniecz** [colouring-cities/colouring-britain#149]

**Comment by polly64** [comment]
@mdsimpson42 did we deal with this one?

**Comment by polly64** [comment]
@mdsimpson42 yes can we trying this on staging . Could you also chat to @matkoniecz re how staging site can work for Loughborough and stiff you're testing - @matkoniecz had some thoughts

**Comment by mdsimpson42** [comment]
Add "yes" and "no" to all such fields

**Comment by mdsimpson42** [comment]
@matkoniecz Polly requested to revisit this as part of the Loughborough consultation

**Comment by mdsimpson42** [comment]
I think we should be consistent and pick a single type of menu for these options. I'm inclined to use the one where you can choose "yes" or "no", rather than the one where you can only choose "yes".

There's another issue, which is that radio buttons are showing up on both these menus, which they weren't before. 
- Confirmed to be happening on [ColouringBritain.org](https://colouringbritain.org/) in Chrome, Edge and Firefox.
- Also confirmed to be happening with my local version of the core platform.

If there are no objections (@polly64, @matkoniecz), I will change all of these menus to use the same yes/no selector (as shown in the first image) and try to fix the issue with the radio buttons.

![Image](https://github.com/user-attachments/assets/6e3380bb-6457-4b74-9c08-4ce73b2a7eb6)
![Image](https://github.com/user-attachments/assets/4772d4bf-e4b7-497e-b067-42aed5dd4630)

**Comment by matkoniecz** [comment]
first style seems more clear to me - I am definitely not opposed

**Comment by mdsimpson42** [comment]
[like]  Mike Simpson reacted to your message:
________________________________

**Comment by polly64** [comment]
yes agree

**Comment by mdsimpson42** [comment]
I've removed the fields that use the "LogicalDataEntryYesOnly".

How do we feel about these ones on the Community page, where you tick the box and then more options appear? I feel like these ones are OK...

![Image](https://github.com/user-attachments/assets/2ebdd55d-0cd3-4659-ac63-ebd03d75b2d4)

![Image](https://github.com/user-attachments/assets/28665e10-32fb-47b6-9b0c-44f22f9bb8a2)

**Comment by polly64** [comment]
yes these work well for me
